habits_styles.css

```diff
@@ -148,25 +150,344 @@ section {
     transition: 0.3s opacity 0ms linear;
 }
 
-
 .brand-image__background {
-    height: 680px;
-    width: 640px;
+    height: 70vh;
+    width: 32vw;
     border-radius: 360px 360px 90px 90px;
     border: 32px solid #f9f9f9;
     background-image: linear-gradient(to right bottom, #eeeeee, #bbbbbb);
     position: absolute;
     top: 260px;
     right: 160px;
     box-shadow: 0 40px 100px #bbbbbb;
+    transition: 0.3s all 0ms linear;
 }
 
 .background__mainscreen {
     height: 1000px;
     width: 100vw;
-    background-image:linear-gradient(to right bottom, #f7f7f7 70%, #bbbbbb);
+    background-image: linear-gradient(to right bottom, #f7f7f7 70%, #bbbbbb);
     position: absolute;
     top: 0;
     z-index: -1;
 }
 
+@media screen and (max-width: 1600px) and (min-width: 1480px) {
+    .brand-image__background {
+        border: 30px solid #f9f9f9;
+        /*top: 340px;*/
+        transition: 0.3s all 0ms linear;
+    }
+}
+
+@media screen and (max-width: 1200px) and (min-width: 1080px) {
+
```

Comment:
Better to make different media queries for different "components"
For example: for navigation, for logo, etc